model/project.rb

```diff
@@ -98,4 +98,5 @@ def self.feature_flag(*flags)
   end
 
   feature_flag :postgresql_base_image
+  feature_flag :gpu_runners
```

Comment:
I think this flag should be named as enable_gpu_runners, to match with rest of the code.